src/main/java/org/takes/facets/auth/social/PsGoogle.java

```diff
@@ -195,7 +195,12 @@ private static Identity parse(final JsonObject json) {
         } else {
             props.put("picture", image.getString("url", "#"));
         }
-        props.put("name", json.getString("displayName", "unknown"));
+        if (json.containsKey("displayName")
```

Comment:
@raphaelln I don't think this solves anything. as far as I understand, this method `getString(String,String)` prevents this NULL problem already. if such a property is not there, a default value will be returned (`"unknown"`)